pages/privacy.tsx

```diff
@@ -235,223 +131,117 @@ export default function Privacy() {
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
+        <h2>Changes to Our Privacy Statement(s)</h2>
+        <p className="para">
           We may update this Privacy Statement periodically. If we make material changes to the way we process your
           Personal Information, we will provide you notice via our service or by some other means, such as via email.
           Please review any changes carefully. If you agree with the changes, continue to use our Services. If you
           object to any of the changes and no longer wish to use our services, you may close your account(s). All
           changes are effective immediately upon posting and your use of our Service after a notice of material change
           or posting of an updated Privacy Statement shall constitute your consent to all changes.
         </p>
-        <div className="font-semibold ml-1 mt-8">Other Important Privacy Information</div>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          If you have questions or concerns regarding this Privacy Statement, our security practices, or our information
-          processing practices please contact us:
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          In writing: Attn: Privacy, 4, Broad Street, Kolkata 700 019, India.
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          Call us at +91-33-22810648.
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8"
-        >
-          Email us at privacy@wallora.com.
-        </p>
-        <p
-          style={{
-            font: '100% Roboto,"Segoe UI",Helvetica;',
-            color: '#261919',
-          }}
-          className="ml-1 mt-8 mb-12"
-        >
-          If you have an unresolved privacy or data use concern that we have not addressed satisfactorily, please email
-          us at privacy@wallora.com
-        </p>
+        <h2>Other Important Privacy Information</h2>
+        <div>
+          <p className="para">
+            If you have questions or concerns regarding this Privacy Statement, our security practices, or our
+            information processing practices please contact us:
+          </p>
+          <p className="para">In writing: Attn: Privacy, 4, Broad Street, Kolkata 700 019, India.</p>
+          <p className="para">Call us at +91-33-22810648.</p>
+          <p className="para">Email us at privacy@wallora.com.</p>
+          <p className="para mb-12">
```

Comment:
instead of adding mb-12 here add to the parent div so that we can add more para later if needed